src/metrics.rs

```diff
@@ -98,6 +97,14 @@ struct Stats {
     fds: usize,
 }
 
+fn get_ticks_per_second() -> Result<f64> {
+    // Safety: This code is taken directly from sysconf
+    match unsafe { libc::sysconf(libc::_SC_CLK_TCK) } {
+        -1 => Err("Clock Tick unsupported".into()),
+        ret => Ok(ret as f64),
+    }
+}
+
```

Comment:
This was taken from sysconf directly

Reply:
I notice we're missing the intermediate conversion to `isize` in `sysconf`'s version, but I don't think that makes any difference, right?

Reply:
in libc it's essentially a isize, but instead of using isize it does conditional compilation so that if it's 64 bit it's i64.

So essentially they are doing what the compiler does with isize anyways.

So the return value is i64 or i32 either of which has an `as` conversion to f64 either way, so it's fine.